### PR TITLE
fix(docs): Hide the `log_namespace` option

### PR DESCRIPTION
**Pull request by jszwedko**
Hiding this field until we are ready to release it.

Signed-off-by: Jesse Szwedko <jesse.szwedko@datadoghq.com>

### CODE REVIEW

**Comment by jszwedko** [comment]
@tobz hoping to get your thoughts on this next week. I verified that `docs::hidden` does end up in the generated schema, but the script to generate cue still seems to be outputting it.

**Comment by tobz** [comment]
Hmm, it looks like you've added this to a `schema` field, not any sort of `log_namespace` field? 🤔

**Comment by jszwedko** [comment]
> Hmm, it looks like you've added this to a `schema` field, not any sort of `log_namespace` field? 🤔

Doh, sorry, I mean to hide all of the `schema` fields, including `log_namespace` 😅 . At first I was just trying to hide that one, but then I realized the entire `schema` field should be hidden until we release it.

**Comment by tobz** [comment]
> Doh, sorry, I mean to hide all of the `schema` fields, including `log_namespace` 😅 . At first I was just trying to hide that one, but then I realized the entire `schema` field should be hidden until we release it.

Ah, gotcha. Yeah, taking a peek, it does seem like this should be doing what you intend. I'll definitely take a look on Monday.

File: src/sources/amqp.rs [hunk 1]
```diff
@@ -74,6 +74,7 @@ pub struct AmqpSourceConfig {
     pub(crate) offset_key: String,
 
     /// The namespace to use. This overrides the global setting.
```
Comment (spencergilbert) [inline src/sources/amqp.rs:76]:
```suggestion
    /// The namespace to use for logs. This overrides the global setting.
```

Comment (jszwedko) [reply src/sources/amqp.rs]: [ef97a2e](https://github.com/vectordotdev/vector/pull/15120/commits/ef97a2ed41e1f34012eae7b8e8438b82c84a9709)

**Review by spencergilbert — APPROVED** [review]

**Comment by jszwedko** [comment]
Merging since test is flakey.

**Pull request merged**